src/EmotiBitPacket.cpp

```diff
@@ -79,12 +80,17 @@ const char* EmotiBitPacket::PayloadLabel::CONTROL_PORT = "CP\0";
 const char* EmotiBitPacket::PayloadLabel::DATA_PORT = "DP\0";
 const char* EmotiBitPacket::PayloadLabel::RECORDING_STATUS = "RS\0";
 const char* EmotiBitPacket::PayloadLabel::POWER_STATUS = "PS\0";
+const char* EmotiBitPacket::PayloadLabel::LSL_MARKER_DESTINATION_TIMESTAMP = "TSC\0";
```

Comment:
All good points. THinking about it abit,
1. `LSL_LOCAL_CLOCK_TIMESTAMP` and `LSL_MARKER_DST_TIMESTAMP / LSL_MARKER_LOCAL_TIMESTAMP`  live on the same domain (of local clock).
2. exploring the namespace
  - `LC` makes sense for local_clock 
  - `LM` makes sense for LSL_MARKER (since it does represent a unique domain - marker source)
  - `LD` can be a good choice to represent LSL MARKER DESTINATION time (which lives in the LC domain)
    - I think using `LL` (LSL LOCAL)  could be confusing, as it may take away from the meaning of LC
    - and i think using `TD` is incorrect, since we have historically using a leading `T` to represent a "domain", not a payload type
    - `MD` is too vague, as it does not represent anything about "LSL". What happens when we have a marker of a different type?